src/joyride/sci.cljs

```diff
@@ -46,10 +69,27 @@
                            (symbol? namespace)
                            (source-script-by-ns namespace)
                            (string? namespace) ;; node built-in or npm library
-                           (if (= "vscode" namespace)
+                           (cond
+                             (= "vscode" namespace)
                              (do (sci/add-class! @!ctx 'vscode vscode)
                                  (sci/add-import! @!ctx (symbol (str @sci/ns)) 'vscode (:as opts))
                                  {:handled true})
+
+                             (active-extension? namespace)
+                             (let [[module-name module] (extension-module namespace)
+                                   ns-sym (symbol (str @sci/ns))
+                                   refer (:refer opts)]
+                               (sci/add-class! @!ctx (symbol namespace) module)
```

Comment:
I suggest we use `(symbol (munge ...))` since `'ext://foo.bar` is not a valid symbol (although CLJS still accepts this, but doesn't check this)

Reply:
Another small feedback:

Can you upgrade to SCI commit `37722fb5e49970fe428781b287f478d8310fd3df` and use `ns` from the map passed to `load-fn`? This is a symbol of the current ns. I think that will be safer to use once we migrate to `sci.async`. I also added `libname` which is the same as `namespace` but a better name, so I'd like to migrate to that.